choose/options.go

```diff
@@ -1,10 +1,13 @@
 package choose
 
-import "github.com/charmbracelet/gum/style"
+import (
+	"github.com/charmbracelet/gum/style"
+	common "github.com/charmbracelet/gum/timeout"
```

Comment:
We can probably leave this as `timeout` import I believe. `timeout.Options` is nice as the option unless there is a reason I am missing as to why it is named as such.

Reply:
yes. common is wrong. Agreed. But as said above, timeout.Options clashes with the choose commands Options struct

Reply:
changed

Reply:
Thank you!